deferral-2-0/Agile-Politico
# Ft admin accept candidature 164774496

**What does this PR do?**
This PR allows the admin user to accept the user as a candidate

**Description of Task to be completed?**
- [x] Create user accept candidate endpoint

**How should this be manually tested?**
After cloning the repository to your local machine - check
[README.md](README.md) for instructions. Using
[Postman](https://www.getpostman.com/) open create a request with this
header
`Key Content-Type: application/json`

What are the relevant pivotal tracker stories?
164774496
